wikiHandler.js

```diff
@@ -86,10 +93,46 @@ function formatPrice(price) {
     }
 }
 
+function parsePage(page, server) {
```

Comment:
This is duplicated code now -- now that it's a function, can we clean that up below?